app/controllers/shared/api.php

```diff
@@ -6,12 +6,13 @@
 use Appwrite\Event\Delete;
 use Appwrite\Event\Event;
 use Appwrite\Event\Mail;
+use Appwrite\Extend\Exception;
```

Comment:
Would it make sense to have this shutdown method in a dedicated controller or under the graphql controller?

Reply:
Yep makes sense in the graphql controller to me